docs/self-managed/installation-methods/helm/configure/secret-management.md

```diff
@@ -192,6 +192,15 @@ webModelerPostgresql:
 
 For additional details on Identity secrets during installation, visit the [installation guide](/self-managed/installation-methods/helm/install.md#create-identity-secrets).
 
+## Document Store secrets
+
+Document Store configuration uses a different pattern that doesn't follow the unified secret structure. These secrets are configured using direct `existingSecret` references with multiple key specifications for different credential components.
```

Comment:
I would just say here:

`Document store secrets are configured using direct ...`

(not mentioning it doesnt follow the unified secret structure directly ;) )